feature-libs/asm/core/facade/asm.service.ts

```diff
@@ -14,7 +17,20 @@ import { AsmSelectors } from '../store/index';
   providedIn: 'root',
 })
 export class AsmService {
-  constructor(protected store: Store<StateWithAsm>) {}
+  constructor(
```

Comment:
facade codes need to be organized with the code in this PR https://github.com/SAP/spartacus/pull/15776/files. Either create 2 facades `asm facade` and `customers facade`, or just one facade`asm facade`.